app/konnect/org-management/audit-logging/reference.md

```diff
@@ -57,23 +78,29 @@ trace_id=8809518331550410226
 user_agent=grpc-node/1.24.11 grpc-c/8.0.0 (linux; chttp2; ganges)
 ```
 
-Each authorization log entry contains the following:
+Each authorization log entry contains the following attributes in addition to the defaults above:
 
 Property | Description
 ---------|-------------
-Timestamp | Time and date of the event in UTC.
-`service_name` | The name of the component within {{site.konnect_short_name}}. For example, `portals` or `runtimegroups`. In the above example, this appears as `Authz.portals`.
-`rt` | Milliseconds since Unix epoch.
-`src` | The IP address of the request originator.
 `action` | The type of action the user performed on the resource. For example, `retrieve`, `list`, or `edit`.
```

Comment:
question: In CEF, the resource type being authorized is in the `Name` field (6th column of the entry). What is the best way to document this, esp. since we'll be supporting additional format(s) in the future?

Reply:
Can we do something like:

```
`Name` <br> *(CEF format only)* | The resource being authorized.
`resource` <br> *(JSON format only)* | The resource being authorized.
```
(with a completely made up name for JSON). 

_Or_, are you saying that the field is the same but appears in different places in CEF vs JSON?

Reply:
A little bit of both. Here's a snippet from the authorization log sample:

`konghq.com CEF:0|ExampleOrg|Konnect|1.0|konnect|Authz.portals|1|rt=16738287345642 action=retrieve...`

The `name` field is determined by position: The 6th pipe-delimited element, `Authz.portals` tells us this is an authorization check against "portals". The `action` is a separate field in the extensions portion of the message. Put it all together, and we know this is a check whether the `principal` can `retrieve` `portals`. In JSON, it might look more like

`{"resource_type":"portals", "action":"retrieve"}`

@cecilmoore please doublecheck my work here.

Reply:
A little bit of both. Here's a snippet from the authorization log sample:

konghq.com CEF:0|ExampleOrg|Konnect|1.0|konnect|Authz.portals|1|rt=16738287345642 action=retrieve...

The name field is determined by position: The 6th pipe-delimited element, Authz.portals tells us this is an authorization check against "portals". The action is a separate field in the extensions portion of the message. Put it all together, and we know this is a check whether the principal can retrieve portals. In JSON, it might look more like

{"resource_type":"portals", "action":"retrieve"}

@cecilmoore please doublecheck my work here.